community/tpm2-tools/APKBUILD

```diff
@@ -0,0 +1,40 @@
+# Contributor:
+# Maintainer: Alexander Sack <asac@pantacor.com>
+pkgname="tpm2-tools"
+pkgver="4.0"
+pkgrel=0
+pkgdesc="TPM2 Tools"
+url="https://github.com/tpm2-software/tpm2-tools/"
+arch="all"
+license="BSD-2-Clause"
+depends=""
+makedepends="libtpm2-tss-dev automake autoconf autoconf-archive libtool openssl-dev doxygen linux-headers curl-dev"
+install=""
+subpackages=""
+source="${pkgname}-${pkgver}.tar.gz::https://github.com/tpm2-software/tpm2-tools/tarball/4.0-rc2"
+builddir="$srcdir/"tpm2-software-tpm2-tools-"*"
+
+build() {
+	echo $builddir
+	cd $builddir
+	./bootstrap
+	./configure \
+		--build=$CBUILD \
+		--host=$CHOST \
+		--prefix=/usr
+	make	
+}
+
+check() {
+	# Replace with proper check command(s)
```

Comment:
Please follow this comment ^

Reply:
i removed the check section for now. there seems to be a unit test failing which i will follow up with upstream.